### PR TITLE
flip: init at 1.2

### PR DESCRIPTION
**Pull request by zmitchell**
## Description of changes

This package is developed by Nvidia to detect subtle changes in rendered images. It can be used in test suites for graphics applications.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/by-name/fl/flip/package.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+stdenv.mkDerivation {
+  pname = "flip";
+  version = "1.2";
+
+  src = fetchFromGitHub {
+    owner = "NVlabs";
+    repo = "flip";
+    rev = "8303adb2060d69423d040453995f4ad1a030a1cc";
+    sha256 = "sha256-jSB79qOtnW/cjApIDcLRqGabnzCIwS7saA+aF1TcyV0=";
```
Comment (robert-manchester) [inline pkgs/by-name/fl/flip/package.nix:16]:
```suggestion
    hash = "sha256-jSB79qOtnW/cjApIDcLRqGabnzCIwS7saA+aF1TcyV0=";
```

**Review by ghost — APPROVED** [review]
LGTM

Result of `nixpkgs-review pr 262183` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>flip</li>
  </ul>
</details>

**Comment by robert-manchester** [comment]
i think the commit message should be `flip: init at 1.2` rather than `init: flip at 1.2` according to the [commit conventions](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#commit-conventions)

File: pkgs/by-name/fl/flip/package.nix (hunk 1 above)
Comment (zmitchell) [reply pkgs/by-name/fl/flip/package.nix]: Fixed

**Review by ghost — APPROVED** [review]

**Pull request merged**